misc/install-deps.sh

```diff
@@ -9,6 +9,12 @@ fi
 
 OPT="${@}"
 
+if [ ! -f /etc/os-release ]; then
+    echo "Your distribution is not supported, so please install pacakges manually."
+    echo
```

Comment:
We don't need this empty `echo`. Instead, please add `\n` at the end of the previous message. The same thing can be applied to the above root permission check.

Reply:
Thanks for your feedback, just amended the commit!